backend/pkg/dbschemas/sql/mysql/queries/system.sql

```diff
@@ -214,6 +215,7 @@ SELECT
    c.TABLE_NAME AS table_name,
    c.COLUMN_NAME AS column_name,
    c.COLUMN_TYPE AS data_type,
```

Comment:
this is kinda weird. the above query we are using c.data_type but this one we are using c.column_type which is different. I didnt want to update it at the expense of potentially causing a bux.